app/assets/javascripts/components/overview/overview_handler.jsx

```diff
@@ -47,13 +50,29 @@ const Overview = createReactClass({
 
   componentDidMount() {
     ServerActions.fetch('timeline', this.props.course_id);
+    this.timeout = poll();
```

Comment:
@AmitJoki shouldn't this be `this.poll()`. If you see the Travis log, all the failing specs are caused by this statement.

Reply:
aaarrrgggggh!

Reply:
Yeah, it should be smh :|